app/code/core/Mage/Catalog/Helper/Image.php

```diff
@@ -642,7 +642,7 @@ public function validateUploadFile($filePath)
         }
 
         if ($imageInfo[0] > $maxDimension || $imageInfo[1] > $maxDimension) {
-            Mage::throwException($this->__('Disalollowed file format.'));
```

Comment:
this probably breaks several translations and should have only get changed in the translation file

Reply:
I've been arguing for the same thing for long time and I was alone saying that we should only update the translation files. At that time literally everybody was against my (and now your) idea. So please be more vocal next time. The people decided that they want to keep in sync strings in php files and in CSV files so now it is what it is.

Reply:
In that line it is about an exception that will be written in a log file. I think that these errors can be corrected in the PHP files and if they exist (although I doubt it) in the CSV files too. A programmer should know English, as well as a system administrator who has access to these files. On the other hand, although I do not agree to modify only the CSV files, I accepted PRs where such modifications were made, because it is for user interface and we should take care of translation without requiring efforts from those who periodically update the source code.. 

There are discussions on this topic in which the administrator takes note of the changes from one version to another and he makes the translations. From the programmer's perspective, honestly speaking, I don't like to see inconsistencies between sentences, especially when I look into the source code. I see something in the interface in English then in the code it's completely different or worse spelling mistakes like the one above, extremely annoying.

Reply:
The important thing is that we do not keep discussing the same things over and over again. I'm fine with the decision taken although it was not the choice I'd love, sometime it's a personal taste and it is fine :-)

Reply:
1. i always said change only translation string in csv
2. @Flyingmana this PR was just to get rid of differences between branches. If we change it back, we should also revert in v20.